cves/CVE-2017-15706.yml

```diff
@@ -10,15 +10,19 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: 
```

Comment:
I believe we should only have one present. Should just go with the more fitting one.

Reply:
resolved.